tests/common-tests.h

```diff
@@ -3227,6 +3227,18 @@ TEST_CASE_METHOD(common_tests, "NULL with optional", "[core][boost][null]")
             CHECK(omi1.is_initialized() == false);
             CHECK(omi2.is_initialized());
             CHECK(omi2.get().get() == 125);
+
+
+            // test us bind with const vector<boost::optional>
+            {
+                const std::string query = "insert";
```

Comment:
This is a bit weird, what do the duplicated statements using this string instead of the literal constant supposed to check?

Reply:
I took so test from "soci/tests/empty/test-empty.cpp". I mean about duplicated statements.
I think aim is check char* and std::sting with operator << work correct and after possible bind values.
My aim is only check bind with vector<boost<optional>> with supported type and CONST vector<boost<optional>> with supported type.
I you want i can delete part of checks?